client/src/ui/organisms/placement/index.tsx

```diff
@@ -451,19 +459,21 @@ function RenderBottomBanner({
         >
           Mozilla ads
         </a>
-        <a
-          className="no-pong"
-          data-glean={
-            "pong: " + (user?.isSubscriber ? "pong->settings" : "pong->plus")
-          }
-          href={
-            user?.isSubscriber
-              ? "/en-US/plus/settings?ref=nope"
-              : "/en-US/plus?ref=nope#subscribe"
-          }
-        >
-          Don't want to see ads?
-        </a>
+        {showNoAds && (
+          <a
+            className="no-pong"
+            data-glean={
+              "pong: " + (user?.isSubscriber ? "pong->settings" : "pong->plus")
+            }
+            href={
+              user?.isSubscriber
+                ? "/en-US/plus/settings?ref=nope"
+                : "/en-US/plus?ref=nope"
```

Comment:
What's the reasoning behind removing the `#subscribe` anchor?

Reply:
Yes, product decision.